roles/keystone-setup/tasks/main.yml

```diff
@@ -97,3 +97,7 @@
 
 - name: restart keystone api
   service: name=keystone state=restarted
+
+- include: federation.yml
+  when: keystone.federation.enabled|bool
+  tags: [ 'keystone-federation']
```

Comment:
We'll have to think about this one, if we want to ever add federation after a stack has been built. We were trying to setup the keystone-setup role so that it only ever executed once for a stack, given the headache we have to go through around admin_token and whatnot. So if we ever want to adjust federation or add more federations, we'd want this stuff to trigger. Will have to consider that in the grands scheme of the role.